From 948798840705154b259f9663ee930b171b02fffd Mon Sep 17 00:00:00 2001
From: Yotam Ben Dosa <ybendosa@habana.ai>
Date: Fri, 11 Feb 2022 14:00:38 +0200
Subject: [PATCH] [SW-72071] board: hl: ami: change inventory path

hls-inventory header has been moved, and thus its header path needs to
be updated.

This resolves:
src/platform/bmc/fwl-platform-oam.cpp:33:27: fatal error:
hls-inventory.h: No such file or directory

Change-Id: I5972ff4a68e3dccd6173f9aa4fcddc8ee3c430ff
Signed-off-by: Yotam Ben Dosa <ybendosa@habana.ai>
---

diff --git a/board/habanalabs/common/post-ami-build.sh b/board/habanalabs/common/post-ami-build.sh
index e1b05ad..8b6dd91 100755
--- a/board/habanalabs/common/post-ami-build.sh
+++ b/board/habanalabs/common/post-ami-build.sh
@@ -21,7 +21,7 @@
 HLMI_DIR=${EMBEDDED_DIR}/hlmi
 LIB_SRC_DIR=${OUTPUT_DIR}/host/arm-buildroot-linux-gnueabi/sysroot/usr/lib
 LIB_INC_DIR=${OUTPUT_DIR}/host/arm-buildroot-linux-gnueabi/sysroot/usr/include
-HLS_UTILS_DIR=${EMBEDDED_DIR}/hls-utils
+HLS_INVENTORY_DIR=${EMBEDDED_DIR}/hls-utils/hls-inventory
 SPECS_SEARCH_DIR=(
 	${SPECS_ROOT}
 	${SPECS_EMBEDDED_ROOT}
@@ -241,7 +241,7 @@
 		copy_header_files
 	done
 
-	cp ${HLS_UTILS_DIR}/include/hls-inventory.h ${BMC_FW_LOADER_INC_DIR}
+	cp ${HLS_INVENTORY_DIR}/include/hls-inventory.h ${BMC_FW_LOADER_INC_DIR}
 
 	# copy habanalabs eeprom, hlmi  and 3rd party headers
 	find_and_copy_headers_from_libs "${BMC_FW_LOADER_INC_DIR}" "${EMBEDDED_DIR}" "eeprom-util" "${EEPROM_HDR_LIST[@]}"
diff --git a/package/hl-fw-loader/Makefile b/package/hl-fw-loader/Makefile
index 2ce8485..d562845 100644
--- a/package/hl-fw-loader/Makefile
+++ b/package/hl-fw-loader/Makefile
@@ -16,7 +16,7 @@
 INCLUDE += -I ./src/platform/bmc
 INCLUDE += -I ./src/platform/bmc/cpld
 INCLUDE += -I ./src/transport
-INCLUDE += -I ${HABANA_SOFTWARE_STACK}/embedded/hls-utils/include
+INCLUDE += -I ${HABANA_SOFTWARE_STACK}/embedded/hls-utils/hls-inventory/include
 
 LZ4_SRC_PATH ?= src
 SRC := $(wildcard src/*.cpp)
